openerp/openupgrade/openupgrade.py

```diff
@@ -135,6 +137,31 @@ def load_data(cr, module_name, filename, idref=None, mode='init'):
 table_exists = openupgrade_tools.table_exists
 
 
+def copy_columns(cr, column_spec):
+    """
+    Copy table columns. Typically called in the pre script.
+
+    :param column_spec: a hash with table keys, with lists of tuples as \
```

Comment:
For docstring, you don't need `\`. You just need to indent a little more (two spaces is the usual), the second line.